pyrightconfig.stricter.json

```diff
@@ -59,6 +59,7 @@
         "stubs/ldap3",
         "stubs/Markdown",
         "stubs/mysqlclient",
+        "stubs/nanoid",
```

Comment:
This is only necessary if some things in the stub are still missing type annotations. Your stubs are already complete, so this is unnecessary :)

```suggestion
```

Reply:
Ah, thanks!

Reply:
yeah that was just me being sloppy, nice test suite caught it though!